Documentation/network/multicast.rst

```diff
@@ -0,0 +1,106 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _enable_multicast:
+
+**********************************
+Multicast Support in Cilium (Beta)
+**********************************
+
+.. include:: ../beta.rst
+
+The multicast capability allows user application to evolve distributing data feeds to
+multiple consumers in the Kubernetes cluster.
+The container network multicast transmission technology based on eBPF focuses on solving
+the problem of efficient multicast transmission in the container network and provides
+support for multiple multicast protocols.
+
+This document explains how to enable multicast support and configure Cilium and CiliumNode
+with multicast group IP addresses and subscribers.
+
+- Requirements (dependencies)
+
+  This guide assumes that Cilium has been correctly installed in your
+  Kubernetes cluster. Please see :ref:`k8s_quick_install` for more
+  information. If unsure, run ``cilium status`` and validate that Cilium is up
+  and running. This guide also assumes Cilium is configured with vxlan mode,
+  which is required when using multicast capability.
```

Comment:
```suggestion
Prerequisites
=============

This guide assumes that Cilium has been correctly installed in your
Kubernetes cluster. Please see :ref:`k8s_quick_install` for more
information. If unsure, run ``cilium status`` and validate that Cilium is up
and running. This guide also assumes Cilium is configured with vxlan mode,
which is required when using multicast capability.
```
Could you give this section a header? Sorry, my previous comment about it was a bit unclear.